srcpkgs/PikoPixel.app/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'PikoPixel.app'
+pkgname=PikoPixel.app
+version=1.0.b9e
+revision=1
+archs="x86_64"
+create_wrksrc=yes
+build_wrksrc=PikoPixel.Sources.${version%.*}-${version##*.}/PikoPixel
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="gnustep-make gcc-objc tar"
+makedepends="gnustep-base-devel gnustep-gui-devel"
+depends="gnustep-base gnustep-gui gnustep-back"
+short_desc="GNUstep application for drawing/editing pixel-art"
+maintainer="Kira Patton <roundduckman@protonmail.com>"
+license="AGPL-3.0"
+homepage="http://twilightedge.com/mac/pikopixel/"
+distfiles="http://twilightedge.com/downloads/PikoPixel.Sources.${version%.*}-${version##*.}.tar.gz"
+checksum=96977fc51343d294c7d7e76d8f1ac7aa82ae9da1d7d082dee6cee8035959afbe
+pre_build() {
+	source /usr/share/GNUstep/Makefiles/GNUstep.sh
+}
+pre_install() {
+	source /usr/share/GNUstep/Makefiles/GNUstep.sh
```

Comment:
Judging from the dir and file name it looks like a build generator script: it's best that you only leave the `pre_build()` with this script.

Reply:
if I remember right I tried to do it with only pre_build but it failed, but I'll check again just in case if it's just me being a dummy :p

Reply:
`=> PikoPixel.app-1.0.b9e_1: running do_install ...
GNUmakefile:382: /application.make: No such file or directory
make: *** No rule to make target '/application.make'.  Stop.
=> ERROR: PikoPixel.app-1.0.b9e_1: do_install: '${make_cmd} STRIP=true PREFIX=/usr DESTDIR=${DESTDIR} ${make_install_args} ${make_install_target}' exited with 2
=> ERROR:   in do_install() at common/build-style/gnu-makefile.sh:40
katei$ `

yep as I expected, it failed without having the preinstall entry too

Reply:
that said, I'm gonna test it with a new void-packages clone, just in case™

Reply:
Even with that it fails so yea, the problem is that the build helper is dropped by the next phase in creating the package, the install phases. As a result that same helper has to be called twice, as ugly as that sounds.